ethereum/contracts/relayer/coreRelayer/CoreRelayerDelivery.sol

```diff
@@ -488,17 +502,17 @@ abstract contract CoreRelayerDelivery is CoreRelayerBase, IWormholeRelayerDelive
     }
   }
 
-  function decodeCancelled(
+  function decodeExecuteInstructionError(
     bytes memory revertData
-  ) private pure returns (uint32 gasUsed, uint256 available, uint256 required) {
+  ) private pure returns (uint32 gasUsed, bool knownError) {
     uint offset = 0;
     bytes4 selector;
     (selector, offset) = revertData.asBytes4Unchecked(offset);
```

Comment:
Do we know that the buffer has at least 4 bytes at this point? We might want to check for the entire exact size or a lower bound for it.

Reply:
Ah good point. Fixed!

Reply:
Ah yes oops, corrected!